Running in wrapper mode for '10'

Running python job.

[name: "CPU-QUOTA"
int_val: 75
int_val: 93
int_val: 59
int_val: 29
int_val: 59
int_val: 43
int_val: 44
, name: "DISK"
int_val: 16
int_val: 16
int_val: 42
int_val: 21
int_val: 29
int_val: 33
int_val: 18
, name: "MEMORY"
int_val: 40
int_val: 49
int_val: 51
int_val: 17
int_val: 40
int_val: 38
int_val: 32
, name: "NET"
int_val: 14
int_val: 13
int_val: 43
int_val: 53
int_val: 16
int_val: 17
int_val: 60
]
The paramters are {u'CPU-QUOTA': array([75, 93, 59, 29, 59, 43, 44]), u'NET': array([14, 13, 43, 53, 16, 17, 60]), u'DISK': array([16, 16, 42, 21, 29, 33, 18]), u'MEMORY': array([40, 49, 51, 17, 40, 38, 32])}
{'13.57.236.233': ['postgres', 'kibana'], '52.53.237.168': ['node-apt-app', 'kibana'], '13.57.232.190': ['elasticsearch', 'mysql'], '13.56.13.241': ['node-apt-app', 'logstash'], '13.57.58.49': ['haproxy', 'kibana'], '54.153.85.122': ['node-apt-app', 'kibana']}
Setting cpu quota to 110000 out of 250000
STRESSING VM_IP 13.57.236.233 AND CONTAINER fd4563743d87, NET 307200.0
STRESSING VM_IP 13.57.236.233 AND CONTAINER fd4563743d87, DISK 15099494.4
STRESSING VM_IP 13.57.236.233 AND CONTAINER fd4563743d87, MEMORY 1200000000.0
Recovering the memory utilization
Setting cpu quota to 232500 out of 250000
STRESSING VM_IP 13.57.236.233 AND CONTAINER 6cb395f9a474, NET 66560.0
STRESSING VM_IP 13.57.236.233 AND CONTAINER 6cb395f9a474, DISK 13421772.8
STRESSING VM_IP 13.57.236.233 AND CONTAINER 6cb395f9a474, MEMORY 1837500000.0
Recovering the memory utilization
Setting cpu quota to 187500 out of 250000
STRESSING VM_IP 52.53.237.168 AND CONTAINER 72bf70afc99a, NET 71680.0
STRESSING VM_IP 52.53.237.168 AND CONTAINER 72bf70afc99a, DISK 13421772.8
STRESSING VM_IP 52.53.237.168 AND CONTAINER 72bf70afc99a, MEMORY 1500000000.0
Recovering the memory utilization
Setting cpu quota to 232500 out of 250000
STRESSING VM_IP 52.53.237.168 AND CONTAINER 6a93346d70ee, NET 66560.0
STRESSING VM_IP 52.53.237.168 AND CONTAINER 6a93346d70ee, DISK 13421772.8
STRESSING VM_IP 52.53.237.168 AND CONTAINER 6a93346d70ee, MEMORY 1837500000.0
Recovering the memory utilization
Setting cpu quota to 147500 out of 250000
STRESSING VM_IP 13.57.232.190 AND CONTAINER f914a2e03362, NET 220160.0
STRESSING VM_IP 13.57.232.190 AND CONTAINER f914a2e03362, DISK 35232153.6
STRESSING VM_IP 13.57.232.190 AND CONTAINER f914a2e03362, MEMORY 1912500000.0
Recovering the memory utilization
Setting cpu quota to 147500 out of 250000
STRESSING VM_IP 13.57.232.190 AND CONTAINER c6bd985fa324, NET 81920.0
STRESSING VM_IP 13.57.232.190 AND CONTAINER c6bd985fa324, DISK 24326963.2
STRESSING VM_IP 13.57.232.190 AND CONTAINER c6bd985fa324, MEMORY 1500000000.0
Recovering the memory utilization
Setting cpu quota to 187500 out of 250000
STRESSING VM_IP 13.56.13.241 AND CONTAINER f8b10d691b77, NET 71680.0
STRESSING VM_IP 13.56.13.241 AND CONTAINER f8b10d691b77, DISK 13421772.8
STRESSING VM_IP 13.56.13.241 AND CONTAINER f8b10d691b77, MEMORY 1500000000.0
Recovering the memory utilization
Setting cpu quota to 72500 out of 250000
STRESSING VM_IP 13.56.13.241 AND CONTAINER 197dde384659, NET 271360.0
STRESSING VM_IP 13.56.13.241 AND CONTAINER 197dde384659, DISK 17616076.8
STRESSING VM_IP 13.56.13.241 AND CONTAINER 197dde384659, MEMORY 637500000.0
Recovering the memory utilization
Setting cpu quota to 107500 out of 250000
STRESSING VM_IP 13.57.58.49 AND CONTAINER ce162cfce35f, NET 87040.0
STRESSING VM_IP 13.57.58.49 AND CONTAINER ce162cfce35f, DISK 27682406.4
STRESSING VM_IP 13.57.58.49 AND CONTAINER ce162cfce35f, MEMORY 1425000000.0
Recovering the memory utilization
Setting cpu quota to 232500 out of 250000
STRESSING VM_IP 13.57.58.49 AND CONTAINER ecf769a2a794, NET 66560.0
STRESSING VM_IP 13.57.58.49 AND CONTAINER ecf769a2a794, DISK 13421772.8
STRESSING VM_IP 13.57.58.49 AND CONTAINER ecf769a2a794, MEMORY 1837500000.0
Recovering the memory utilization
Setting cpu quota to 187500 out of 250000
STRESSING VM_IP 54.153.85.122 AND CONTAINER f340b9f2855e, NET 71680.0
STRESSING VM_IP 54.153.85.122 AND CONTAINER f340b9f2855e, DISK 13421772.8
STRESSING VM_IP 54.153.85.122 AND CONTAINER f340b9f2855e, MEMORY 1500000000.0
Recovering the memory utilization
Setting cpu quota to 232500 out of 250000
STRESSING VM_IP 54.153.85.122 AND CONTAINER 93ffc048982e, NET 66560.0
STRESSING VM_IP 54.153.85.122 AND CONTAINER 93ffc048982e, DISK 13421772.8
STRESSING VM_IP 54.153.85.122 AND CONTAINER 93ffc048982e, MEMORY 1837500000.0
Recovering the memory utilization
Using 1 trials
Initializing Machines
ab -p post.json -T application/json -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/psql/users/
ab -p post.json -T application/json -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/mysql/users/
Sleeping for 2 seconds
ab -q -n 200 -c 50 -s 9999 -e results_file http://13.57.58.49:80/app/psql/users/ > output0.txt
ab -q -p post.json -T application/json -n 200 -c 50 -s 9999 -e results_file http://13.57.58.49:80/app/psql/users/ > output1.txt
ab -q -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/mysql/users/ > output2.txt
ab -q -p post.json -T application/json -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/mysql/users/ > output3.txt
ab -q -n 800 -c 500 -s 9999 -e results_file http://13.57.58.49:80/app/users/ > output4.txt
ab -n 1 -s 9999 -e results_file http://13.57.58.49:80/app/elastic/users/3 > output5.txt
Please ignore the following new lines (if any)
13.72,3645.608,2208.0,5538.0,8841.0
11.05,4525.459,1918.0,6905.0,13717.0
44.78,11166.673,6142.0,10798.0,13535.0
61.36,8149.183,3595.0,9706.0,9791.0
121.9,4101.849,52.0,1655.0,4824.0
0.22,4561.618,-1,-1,-1
total:253.03,36150.39,15420.33394,36107.33394,52213.33394
Sleeping for 15 seconds for proper deletion
Sleeping for 5 more seconds for proper deletion
Experiment results are {'latency_90': [36107.33394], 'latency': [36150.39000000001], 'rps': [253.03000000000003], 'latency_50': [15420.33394], 'latency_99': [52213.33394]}
Got result 52213.333940

----------------------------------------
Problem running the job:
(<class 'subprocess.CalledProcessError'>, CalledProcessError(), <traceback object at 0x11992e8c0>)
Traceback (most recent call last):
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/runner.py", line 42, in job_runner
    elif job.language == PYTHON: run_python_job(job)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/runner.py", line 148, in run_python_job
    save_job(job)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/helpers.py", line 72, in save_job
    file_write_safe(filename, job.SerializeToString())
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/helpers.py", line 46, in file_write_safe
    sh(cmd)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/helpers.py", line 19, in sh
    subprocess.check_call(cmd, shell=True)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 540, in check_call
    raise CalledProcessError(retcode, cmd)
CalledProcessError: Command 'mv "/var/folders/0x/f3sy_kt92pbfpm7_jg_d1gqm0000gn/T/tmpS6s5NY" "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/bayOptSearch/jobs/00000010.pb"' returned non-zero exit status 1
None
----------------------------------------
Process Process-131:
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/runner.py", line 62, in job_runner
    job      = load_job(job_file)
  File "/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/spearmint/helpers.py", line 76, in load_job
    fh = open(filename, 'rb')
IOError: [Errno 2] No such file or directory: u'/Users/rahulbalakrishnan/Desktop/throttlebot/src/spearmint/bayOptSearch/jobs/00000010.pb'
